boot/zephyr/os.c

```diff
@@ -42,3 +42,10 @@ void os_heap_init(void)
 {
     mbedtls_memory_buffer_alloc_init(mempool, sizeof(mempool));
 }
+
+void exit(int status)
+{
+	/* Why? */
```

Comment:
This is unlikely to be the correct fix :-)

This is a consequence of me moving the allocator to use the mbed_tls allocator. I did this with the RSA code, making a change to the config file, and there likely needs to be a change made to the config file used for ECDSA. I'll try to fix this today.